src/api/rest.js

```diff
@@ -10,13 +10,17 @@ import {
 
 const API_URL = `${process.env.GNOSISDB_URL}/api`
 
+const whitelistedAddressesFilter = Object.keys(process.env.WHITELIST)
+  .map((address, index) => `${index ? '&' : '?'}creator=${address}`)
+  .join('')
+
```

Comment:
Can we use a package for this? like `querystring`?

Reply:
@andre-meyer yes, i will update it

Reply:
But I dont think that it is really needed because for me it is simple enough without any packages

Reply:
It took me a while to understand that this generates a querystring and I think it would be cleaner to use a package to generate querystrings. We'll surely need querystrings again in the future and writing it yourself is errorprone with more complex objects